projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -45,14 +47,15 @@
           {{ node.title }}
         </cx-generic-link>
         <button
-          [attr.tabindex]="depth < 1 ? 0 : -1"
           [attr.aria-haspopup]="true"
           [attr.aria-expanded]="false"
           [attr.aria-label]="node.title"
           (click)="toggleOpen($any($event))"
           (mouseenter)="onMouseEnter($event)"
-          (keydown.space)="toggleOpen($any($event))"
+          (keydown.space)="onSpace($any($event))"
           (keydown.esc)="back()"
+          (keydown.arrowDown)="focusOnNode($any($event))"
```

Comment:
Just curious, if we have `arrowControls` in the class, do we need the `arrowDown` input in the template too?

Reply:
This helps with UX since you can enter the next node by just pressing down, especially on mobile.